test/es/clients_v6_test.go

```diff
@@ -42,7 +42,7 @@ func TestNewClientV6(t *testing.T) {
 	assert.NotEqual(t, res.IsError(), nil, "response with error")
 }
 
-func TestGetAwsClientV6(t *testing.T) {
+func Skip_TestGetAwsClientV6(t *testing.T) {
```

Comment:
the only thing missing here is likely environment variables, shouldn't we be able to easily add them?

Reply:
✅